es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 
 <br>
 
 **11. E-step: Evaluate the posterior probability Qi(z(i)) that each data point x(i) came from a particular cluster z(i) as follows:**
 
-&#10230;
+&#10230; Paso E: Evalúa la probabilidad posterior Qi(z(i)) de que cada punto de datos x(i) provenga de un determinado clúster z(i) de la siguiente manera:
 
 <br>
 
 **12. M-step: Use the posterior probabilities Qi(z(i)) as cluster specific weights on data points x(i) to separately re-estimate each cluster model as follows:**
 
-&#10230;
+&#10230; M-step: Usa las probabilidades posteriores Qi(z(i)) como pesos específicos del clúster en los puntos de datos x(i) para re-estimar por separado cada modelo de clúster de la siguiente manera:
 
 <br>
 
 **13. [Gaussians initialization, Expectation step, Maximization step, Convergence]**
 
-&#10230;
+&#10230; [Inicialización Gaussiana, Etapa de Expectativa, Etapa de Maximización, Convergencia]
 
 <br>
 
 **14. k-means clustering**
 
-&#10230;
+&#10230; Agrupamiento k-means
 
 <br>
 
 **15. We note c(i) the cluster of data point i and μj the center of cluster j.**
 
-&#10230;
+&#10230; Denotamos c(i) el clúster de datos al punto i, y μj el centro del clúster j.
 
 <br>
 
 **16. Algorithm ― After randomly initializing the cluster centroids μ1,μ2,...,μk∈Rn, the k-means algorithm repeats the following step until convergence:**
 
-&#10230;
+&#10230; Algoritmo - Después de haber iniciado aleatoriamente los centroides del clúster μ1,μ2,...,μk∈Rn, el algoritmo k-means repite el siguiente paso hasta la convergencia:
 
 <br>
 
 **17. [Means initialization, Cluster assignment, Means update, Convergence]**
 
-&#10230;
+&#10230; [Inicialización de medias, Asignación de Cluster, Actualización de medias, Convergencia]
 
-<br>
+<br> 
 
 **18. Distortion function ― In order to see if the algorithm converges, we look at the distortion function defined as follows:**
 
-&#10230;
+&#10230; Función de Distorsión - Para ver si el algoritmo converge, observamos la función de distorsión definida de la siguiente manera:
 
 <br>
 
 **19. Hierarchical clustering**
 
-&#10230;
+&#10230; Agrupación Jerárquica
 
 <br>
 
 **20. Algorithm ― It is a clustering algorithm with an agglomerative hierarchical approach that build nested clusters in a successive manner.**
 
-&#10230;
+&#10230; Algoritmo - Es un algoritmo de agrupamiento con un enfoque de aglomeramiento jerárquico que construye clústeres anidados de forma sucesiva.
 
 <br>
 
 **21. Types ― There are different sorts of hierarchical clustering algorithms that aims at optimizing different objective functions, which is summed up in the table below:**
 
-&#10230;
+&#10230; Tipos - Hay diferentes tipos de algoritmos de agrupamiento jerárquico que apuntan a optimizar diferentes funciones objetivo, que se resumen en la tabla a continuación:
```

Comment:
The translation here for "aim" is "_que [tienen por objetivo](https://diccionario.reverso.net/ingles-espanol/aims)_".